File: python/liczby_2_3.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def liczby2(a=10, b=99):
    
    """
    Program drukuje wszystkie liczby dwucyfrowe o niepowtarzających się
    cyfrach oraz zwraca ilość takich liczb
    """
    ile = 0
    
    for i in range(1, 10):
        for j in range(0, 10):
            if i != j:
                print("{}{} ".format(i,j), end='')
                ile += 1
    return ile
			          
        
    
def liczby3(a=100, b=999):
    
    """
    Program drukuje wszystkie liczby trzycyfrowe o niepowtarzających się
    cyfrach oraz zwraca ilość takich liczb
    """
    
    ile = 0
    
    for i in range(1, 10):
        for j in range(0, 10):
            for k in range(0.10):
                if i != j & i != k & j != k:
                    print("{}{} ".format(i,j,k), end='')
                    ile += 1
    return ile

def main(args):
    
    print(liczby3())
    
    return 0

if __name__ == '__main__':
    import sys
sys.exit(main(sys.argv))

File: cpp/liczby2_3.cpp
/*
 * liczby2_3.cpp
 * 
 * Copyright 2018  <>
 */


#include <iostream>
#include <iomanip>
using namespace std;

int liczby2() {
    int ile = 0;
    
    for (int i = 1; i < 10; i++){
        for (int j = 0; j < 10; j++){
            if (i != j) {
            cout << i << j << " ";
            ile++;
            }
            }
            
            cout << endl;
            
        }
    return ile;
}

int liczby3() {
    
    
    }
    

int main(int argc, char **argv)
{
    int ile = liczby2();
	cout << "\n\nLiczb 2-cyfrowych: " << ile << endl;
	return 0;
}

